test/project_types/script/commands/create_test.rb

```diff
@@ -107,7 +107,7 @@ def test_directory_already_exists
 
       private
 
-      def perform_command
+      def perform_command_snake_case
```

Comment:
Why is this done?

Reply:
when we deprecate snake_case on the CLI, our function names will be regular (so just `perform_command`) 😄